mmdet/datasets/coco.py

```diff
@@ -45,7 +45,12 @@ def load_annotations(self, ann_file):
         """
 
         self.coco = COCO(ann_file)
-        self.cat_ids = self.coco.get_cat_ids(cat_names=self.CLASSES)
+        self.cat_ids = []
+
+        # make cat_ids consistent with the order of class_name
+        for class_name in self.CLASSES:
+            self.cat_ids.append(self.coco.get_cat_ids(cat_names=class_name)[0])
```

Comment:
Since we already wrapped coco api in mmdet. Maybe we could add an argument to get some thing like `self.coco.get_cat_ids(sorted=True)`?